schema/artist/index.js

```diff
@@ -238,6 +238,18 @@ const ArtistType = new GraphQLObjectType({
           gravity(`artist/${id}/artworks`, options)
             .then(exclude(options.exclude, 'id')),
       },
+      formatted_artworks_count: {
```

Comment:
super minor but you can do string interpolation (like ruby's `#{..}`):

``` js
totalWorks = `${published_artworks_count} ${published_artworks_count > 1 ? 'works' : 'work')}`
const forSaleWorks = forsale_artworks_count ? `${forsale_artworks_count} for sale` : null
return (forSaleWorks && totalWorks) ? `${totalWorks}, ${forSaleWorks}` : totalWorks;
```

that's actually not really any better now that i look at it.